src/layers/medCore/source/virtualRepresentation/medVirtualRepresentation.cpp

```diff
@@ -790,7 +790,8 @@ void medVirtualRepresentation::removeTooOldEntry()
 QModelIndex medVirtualRepresentation::getModelIndex(medDataIndex index)
 {
     QModelIndex indexRes;
-    QString indexAsString = indexToFileSysPath(index.asString());
```

Comment:
Thanks for correction but a better way consist to modify fonctions signatures by
```
QString indexToFileSysPath(const QString &index); 
```
into medDataHub.h

Reply:
thx, PR updated with the changes